src/devtools/client/webconsole/components/Input/EagerEvalFooter.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { FC, useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
 import { ValueFront } from "protocol/thread";
 import ObjectInspector from "devtools/client/webconsole/utils/connected-object-inspector";
-import { eagerEvaluateExpression } from "../../utils/autocomplete-eager";
+import { eagerEvaluateExpression } from "../../actions/autocomplete-eagerer";
 
 function useEagerEvalPreview(expression: string) {
   const [value, setValue] = useState<ValueFront | null>(null);
+  const dispatch = useDispatch();
+
   const expressionRef = useRef(expression);
 
   useEffect(() => {
```

Comment:
The linter wants `dispatch` to be in the deps array here, which I think should be harmless and technically correct (though probably not functionally important).

Reply:
yeah it's [silly](https://react-redux.js.org/api/hooks#usedispatch) 😬